src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -30,6 +28,17 @@
       <Name>System.IO.FileSystem</Name>
       <Private>True</Private>
     </ProjectReference>
+
+    <!-- 
+      Until an updated packages is published, temporarily copy the locally built System.Runtime.Extensions library
```

Comment:
If the plan is to revert this when an updated package is available, I'd suggest including the issue number here for reference.